apps/fluent-tester/src/FluentTester/TestComponents/ActivityIndicator/ActivityIndicatorTest.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+import { ActivityIndicator } from '@fluentui-react-native/experimental-activity-indicator';
+import { Text } from '@fluentui/react-native';
+import { Stack } from '@fluentui-react-native/stack';
+import { stackStyle } from '../Common/styles';
+import { Test, TestSection, PlatformStatus } from '../Test';
+import { ACTIVITYINDICATOR_TESTPAGE } from './consts';
+
+const activityIndicatorTest: React.FunctionComponent<{}> = () => {
+  /** Bug: Customize doesn't do anything
+   * Tried having tokens not be props, but didn't work
+   * Not sure how to test/check where the tokens are passed through other than just looking at the final render
+   */
```

Comment:
Rather than just "Bug", I would put "GH: ###" with the issue number so that we know where it is tracked. You can also just say the first line "Customize doesn't work" and put more info in the bug.